Sources/DolbyIORTSCore/Model/StreamingStatistics.swift

```diff
@@ -5,39 +5,46 @@
 import Foundation
 import MillicastSDK
 
-public struct StreamingStatistics {
+public struct AllStreamingStatistics : Equatable, Hashable {
+    public let roundTripTime: Double?
+    public var videoStatsInboundRtpList: [StatsInboundRtp]?
+    public var audioStatsInboundRtpList: [StatsInboundRtp]?
+}
+
+public struct StreamingStatistics : Equatable, Hashable {
     public let roundTripTime: Double?
     public let audioStatsInboundRtp: StatsInboundRtp?
```

Comment:
Do you know if `audioStatsInboundRtp` or `videoStatsInboundRtp` are really optional stats for a stream ?

Reply:
yes, we may get statistics without inbound audio or video statistics